pkg/phlare/phlare.go

```diff
@@ -293,11 +295,12 @@ func (f *Phlare) setupModuleManager() error {
 	mm.RegisterModule(QueryFrontend, f.initQueryFrontend)
 	mm.RegisterModule(QueryScheduler, f.initQueryScheduler)
 	mm.RegisterModule(Compactor, f.initCompactor)
+	mm.RegisterModule(Admin, f.initAdmin)
 	mm.RegisterModule(All, nil)
 
 	// Add dependencies
 	deps := map[string][]string{
-		All: {Ingester, Distributor, QueryScheduler, QueryFrontend, Querier, StoreGateway},
+		All: {Ingester, Distributor, QueryScheduler, QueryFrontend, Querier, StoreGateway, Admin},
```

Comment:
@cyriltovena I think we were discussing this and the idea was to run it as a separate workload (pod), right?

Reply:
I think in the long run we should create a new target yes most likely `admin` and run a single pod in each cluster with low resource requests.

However I suspect that for now it's way enough to have with any component (store-gateway ?) that has storage dependencies. This will already be super useful.